code/go/0chain.net/smartcontract/config.go

```diff
@@ -21,11 +21,12 @@ const (
 	String
 	StateBalance
 	Key
+	Cost
 	NumberOfTypes
 )
 
 var ConfigTypeName = []string{
-	"int", "int64", "int32", "time.duration", "float64", "bool", "string", "state.Balance",
+	"int", "int64", "int32", "time.duration", "float64", "bool", "string", "state.Balance", "Cost",
```

Comment:
Use lower case of "cost" to keep the consistency

Reply:
is it the same as Balance and should be started from capital letter?